python/cudf/cudf/core/indexed_frame.py

```diff
@@ -1653,12 +1634,6 @@ def var(
         a    1.666667
         b    1.666667
         dtype: float64
-
-        .. pandas-compat::
-            :meth:`pandas.DataFrame.var`, :meth:`pandas.Series.var`
-
-            Parameters currently not supported are `level` and
```

Comment:
Are `level` and `numeric_only` no longer supported?

Reply:
`level` no long exists as a keyword (including pandas)

`numeric_only` is supported in `DataFrame` reduction ops